### PR TITLE
feat(stage-filmstrip) Added user configurable max

### PR DESCRIPTION
**Pull request by robertpin**
The user can set the max number of participants that can be displayed on stage
Send the number on follow me to all participants

### CODE REVIEW

File: react/features/settings/components/web/MoreTab.js [hunk 1]
```diff
@@ -490,6 +523,54 @@ class MoreTab extends AbstractDialogTab<Props, State> {
         );
     }
 
+    _renderMaxStageParticipantsSelect: () => void;
+
+    /**
+     * Returns the React Element for the desktop share frame rate dropdown.
+     *
```
Comment (horymury) [inline react/features/settings/components/web/MoreTab.js:530]: wrong JSDOC :)

File: react/features/filmstrip/actions.web.js [hunk 2]
```diff
@@ -435,3 +436,16 @@ export function setStageParticipants(queue) {
...
+ * Sets the max number of participants to be displayed on stage.
+ *
+ * @param {number} maxParticipants - Max number of participants.
+ * @returns {Object}
+ */
+export function setMaxStageParticipants(maxParticipants) {
+    return {
+        type: SET_MAX_STAGE_PARTICIPANTS,
+        maxParticipants
+    };
```
Comment (horymury) [inline react/features/filmstrip/actions.web.js:450]: Shouldn't this line consider the current maxParticipants?: 

https://github.com/jitsi/jitsi-meet/blob/42703fed474c72606f21e92214a24a0b1b5aaed7/react/features/filmstrip/actions.web.js#L268

File: react/features/filmstrip/middleware.web.js [hunk 3]
```diff
@@ -120,7 +127,8 @@ MiddlewareRegistry.register(store => next => action => {
         const { dispatch, getState } = store;
         const { participantId, pinned } = action;
         const state = getState();
-        const { activeParticipants } = state['features/filmstrip'];
+        const { activeParticipants, maxStageParticipants } = state['features/filmstrip'];
+        const maxNoOfParticipants = Math.min(MAX_ACTIVE_PARTICIPANTS, maxStageParticipants);
```
Comment (horymury) [inline react/features/filmstrip/middleware.web.js:131]: can maxStageParticipants ever be > than `MAX_ACTIVE_PARTICIPANTS`

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/filmstrip/actions.web.js (hunk 2 above)
Comment (robertpin) [reply react/features/filmstrip/actions.web.js:450]: Not really. There it's used as an absolute max kind of thing, so it's ok to leave it like this

File: react/features/filmstrip/middleware.web.js (hunk 3 above)
Comment (robertpin) [reply react/features/filmstrip/middleware.web.js:131]: No. I moved this check on SET_MAX_STAGE_PARTICIPANTS to make sure that I can't

**Review by horymury — APPROVED** [review]

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Review by horymury — APPROVED** [review]

**Pull request merged**